js/fizzbuzz.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+var FizzBuzz = (function() {
+
+  var array = [];
+  var fizzBuzz = function (firstWord, secondWord) {
+
+    if (firstWord !== '') {
+      this.firstWord = firstWord;
+    }
+    else {
+      this.firstWord = 'Fizz';
+    }
```

Comment:
For a little bit better readability, you might want to use the or (`||`) operator here as part of your variable assignment: `this.firstWord = firstWord || 'Fizz';`